qutebrowser/browser/browsertab.py

```diff
@@ -413,6 +413,13 @@ def selection(self, callback):
     def follow_selected(self, *, tab=False):
         raise NotImplementedError
 
+    def set_user_interacted(self):
```

Comment:
I opted to duplicate this in webengine/webkittab because it dosen't make sense without a functional focustools, which may not be part of future backends initially. I can easily move the implmentation here though if that makes more sense.